vespajlib/src/main/java/com/yahoo/tensor/MixedTensor.java

```diff
@@ -26,17 +26,44 @@ public class MixedTensor implements Tensor {
 
     /** The dimension specification for this tensor */
     private final TensorType type;
+    private final int blockSize; // aka dense subspace size
 
-    /** The list of cells in the tensor */
-    private final List<Cell> cells;
+    static class DenseBlock {
+        final TensorAddress sparseAddr;
+        final double[] cells;
```

Comment:
Do we want coarser numbers here too, later?

Reply:
yes; I would like to have float and byte as options at least.